Formula/treefrog.rb

```diff
@@ -24,7 +24,11 @@ class Treefrog < Formula
   end
 
   depends_on xcode: :build
+  depends_on "cmake"
```

Comment:
Where is this used?

Reply:
needed in the `configure` step

```
==> ./configure --prefix=/opt/homebrew/Cellar/treefrog/2.6.0 --enable-shared-mongoc --enable-shared-glog
cmake: command not found
```

Reply:
Whyyyyyyyyyy. Okay, but let's add a comment that `# ./configure calls cmake`